openedx/core/djangoapps/schedules/management/commands/send_verified_upgrade_deadline_reminder.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import print_function
+
+import datetime
+
+from dateutil.tz import tzutc, gettz
+from django.core.management.base import BaseCommand
+from django.test.utils import CaptureQueriesContext
+from django.db.models import Prefetch
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.db import DEFAULT_DB_ALIAS, connections
+
+from openedx.core.djangoapps.schedules.models import Schedule
+from openedx.core.djangoapps.user_api.models import UserPreference
+
+from edx_ace.message import MessageType
+from edx_ace.recipient_resolver import RecipientResolver
+from edx_ace import ace
+from edx_ace.recipient import Recipient
+
+
+from course_modes.models import CourseMode, format_course_price
+from lms.djangoapps.experiments.utils import check_and_get_upgrade_link
+
+
+class VerifiedUpgradeDeadlineReminder(MessageType):
+    pass
+
+
+class VerifiedDeadlineResolver(RecipientResolver):
```

Comment:
Is this still work-in-progress?  If so, we can get back to this later.  At a first glance, I'm wondering where we filter out users who have already verified?  And filter out courses whose verification deadlines have already passed.